Tests/Parse/ID3TagSizeParserTest.swift

```diff
@@ -21,8 +21,14 @@ struct ID3TagSizeParserTest {
         #expect(ID3TagSizeParser().parse(data: mp3V23) == 245864)
     }
 
+    @Test func parseTagSizeToBeModified() {
+        let mp3 = NSData(contentsOfFile: PathLoader().pathFor(name: "example-to-be-modified", fileType: "mp3"))!
+        #expect(ID3TagSizeParser().parse(data: mp3) < mp3.count)
+    }
```

Comment:
Also this test is not needed, the parser is working fine (as mentioned above probably this is something related to the examples).